src/app/core/entity-components/entity-list/list-filter/list-filter.component.html

```diff
@@ -0,0 +1,33 @@
+<div *ngIf="displayAsToggle">
```

Comment:
The styling is not very good yet

![filters_screenshot](https://user-images.githubusercontent.com/33730997/104454124-a91ac200-55a5-11eb-87d1-f12738def17a.png)